curator/Dockerfile

```diff
@@ -13,7 +13,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM bobrik/curator:5.8.1
+FROM alpine:3.11
```

Comment:
@ryw wdyt to replace to ap-base here too?

Reply:
yeah let's do it :)

Reply:
can we switch this to astronomer/ap-base:3.11.5-3?